.circleci/config.yml

```diff
@@ -516,6 +545,30 @@ jobs:
             - run:
                 name: Run hwacha tests
                 command: .circleci/run-tests.sh hwacha
+    tracegen-run-tests:
```

Comment:
How long do these tests take to run?

Reply:
On CI it takes about 40 minutes to build and 4 minutes to run.